packages/taco-auth/src/providers/eip4361/external-eip4361.ts

```diff
@@ -1,42 +0,0 @@
-import { SiweMessage } from 'siwe';
-
-import { AuthSignature } from '../../auth-sig';
-
-import { EIP4361_AUTH_METHOD } from './common';
-
-export const USER_ADDRESS_PARAM_EXTERNAL_EIP4361 =
```

Comment:
We still need this file - just not this constant.

You still want developers to provide the `SingleSignOnEIP4361AuthProvider` but simply use it  with `:userAddress`.

Reply:
^ Let's just make these changes directly in this PR; mo need for another branch. You can always clean up commits afterwards, within this branch, if there are subsequent issues.

Reply:
Ok 👍 
Pushed the changes to be in this PR. And converted it to a Draft.

Reply:
@derekpierre 
Pushed some changes again according to our discussion.  Is it good now so I finalize it and mark it as ready?
(I will later add some method docs and check the CI failed jobs...)
Thanks,